### ISSUE TITLE
UnusedVariableSniff wrong

### ISSUE THREAD

**Issue by aiphee** [slevomat/coding-standard#1145]
Hi i am using `symplify/easy-coding-standard` with this rule and there is false positive.

```php
        $counter = 0;
        foreach([1, 2, 3]  as $number) {
            echo $number;
            if (++$counter === 2) {
                break;
            }
        }
```

```
 Unused variable $counter.

 Reported by:
"SlevomatCodingStandard\Sniffs\Variables\UnusedVariableSniff.UnusedVariable"
```

**Comment by kukulich** [comment]
Duplicate of #1096
